cln-grpc/proto/node.proto

```diff
@@ -621,10 +625,18 @@ message ListdatastoreDatastore {
 }
 
 message ListinvoicesRequest {
+	// ListInvoices.index
+	enum ListinvoicesIndex {
+		CREATED = 0;
+		UPDATED = 1;
+	}
 	optional string label = 1;
 	optional string invstring = 2;
 	optional bytes payment_hash = 3;
 	optional string offer_id = 4;
+	optional ListinvoicesIndex index = 5;
```

Comment:
Small question here:
Would it also be an option to remove the enum and have:
```
message ListinvoicesRequest {
	optional string label = 1;
	optional string invstring = 2;
	optional bytes payment_hash = 3;
	optional string offer_id = 4;
	optional uint64 start_created = 5;
	optional uint64 start_updated = 6;
	optional uint64 limit = 7;
}
```

Most likely I'm missing something ;-)

Reply:
These are generated, I have no idea: @cdecker ?